care/facility/api/serializers/facility.py

```diff
@@ -104,3 +109,27 @@ class Meta:
     def create(self, validated_data):
         validated_data["created_by"] = self.context["request"].user
         return super().create(validated_data)
+
+
+class FacilityImageUploadSerializer(serializers.ModelSerializer):
+    cover_image = serializers.ImageField(required=True, write_only=True)
```

Comment:
why is this write_only, there is not retrieve API associated to this endpoint right?

Reply:
There is no retrieve API with upload serializer, that's why I'm making it write-only, cover_image_url will be there on facility retrieve API endpoint, am I going in right direction?